src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -893,10 +893,16 @@ export default {
 		 */
 		async handleShouldRefreshConversations(options) {
 			if (options?.token && options?.properties) {
-				await this.$store.dispatch('setConversationProperties', {
-					token: options.token,
-					properties: options.properties,
-				})
+				if (this.$store.getters.isInLobby && options.properties.lobbyState === WEBINAR.LOBBY.NONE) {
+					// If the lobby enabled, non-moderators do not have access to messages
+					// On disabling, should fetch complete room object to set up chat context
+					await this.$store.dispatch('fetchConversation', { token: options.token })
```

Comment:
that's a synchronized denial of service :see_no_evil: 
the readmarker and other values should be there already?

Reply:
No, client doesn't know them yet

Reply:
We can move that up, if it helps to prevent a request per user when the moderator ends the lobby?